minimal/server.go

```diff
@@ -190,6 +191,25 @@ func (t *txpoolHub) GetNonce(root types.Hash, addr types.Address) uint64 {
 	return account.Nonce
 }
 
+func (t *txpoolHub) GetBalance(root types.Hash, addr types.Address) (*big.Int, error) {
+	snap, err := t.state.NewSnapshotAt(root)
+	if err != nil {
+		return nil, fmt.Errorf("unable to get snapshot for root, %v", err)
+	}
+
+	result, ok := snap.Get(keccak.Keccak256(nil, addr.Bytes()))
+	if !ok {
+		return nil, fmt.Errorf("unable to get result from snapshot, %v", err)
+	}
```

Comment:
I think that in this middle condition it makes sense to return zero value, because for non existing accounts we should return 0 AFAIK.
```suggestion
	result, ok := snap.Get(keccak.Keccak256(nil, addr.Bytes()))
	if !ok {
		return big.NewInt(0), nil
	}
```

Reply:
Confirming we have this exact same functionality you've suggested in `eth_endpoint.go` for fetching the account balances:
https://github.com/0xPolygon/polygon-sdk/blob/6d40a0e5108374ef2dfe5ea2d49aaac261779101/jsonrpc/eth_endpoint.go#L439-L443

Fixed 👍 

6d40a0e